src/styles/index.scss

```diff
@@ -1,3 +1,145 @@
 body {
   margin: 0;
 }
+
+.product-card {
+  width: 200px;
+  height: auto;
+
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__img-link {
+    display: block;
+    background: url(../images/imac.jpeg);
+    background-size: 100%;
+    margin: 0 auto;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text-link {
+    display: block;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #060b35;
+    text-decoration: none;
+
+    margin: 40px 0 4px;
+  }
+
+  &__code {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #616070;
+
+    margin: 0;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
```

Comment:
apply only top or only bottom margins to avoid implicit behaviour like margin collapse